<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgMoveTool</class>
 <widget class="QWidget" name="WdgMoveTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>130</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpSelectionMethod">
     <property name="title">
      <string>Selection Mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioSelectedLayer">
        <property name="toolTip">
         <string>Move the layer that you have currently selected in the layerbox with its masks. Shortcut: ctrl-click.</string>
        </property>
        <property name="text">
         <string>Move current layer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioFirstLayer">
        <property name="toolTip">
         <string>Move the first layer with visible content at the place where you click. This will also select that layer in the layerbox.</string>
        </property>
        <property name="text">
         <string>Move layer with content</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioGroup">
        <property name="toolTip">
         <string>Move the group containing the first layer that contains visible content. Shortcut: ctrl-shift-click.</string>
        </property>
        <property name="text">
         <string>Move the whole group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="SpecialSpacer" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
